README.md

```diff
@@ -1,2 +1,40 @@
-# Portfolio
- A mobile Portfolio project to showcase my skills and tell a little about myself
+## Porfolio-mobile
+> This is a portfolio that shows my proficiency as I go through Microverse. It is a work-in-progress and I hope to make it a masterpiece.
+
+
+## Built With
+- HTML
+- CSS
+- Linter
+
+
+## Authors
+
+👤 **Titilope Apuabi**
+
+- GitHub: [@eternalgratis](https://github.com/Eternalgratis)
+- Twitter: [@eternalgratis](https://twitter.com/eternalgratis)
+- Instagram: [@eternalgratis](https://www.instagram.com/eternalgratis/)
+
+
+ 
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## Acknowledgments
+
+- Hat tip to anyone who's code/resource was used
+- Inspiration
```

Comment:
- [ ] Here let's remove these two lines since there's no one to acknowledge currently. Try not to forget in the future though 😄